docusaurus.config.js

```diff
@@ -434,7 +445,27 @@ const config = {
     [
       "@docusaurus/plugin-client-redirects",
       {
-        redirects,
+        redirects: old_site_redirects,
+        createRedirects(existingPath) {
+          if (existingPath.includes("/dev/")) {
```

Comment:
Is there ever a chance of `/dev/` showing up elsewhere in a URL? Can `.startsWith` be used here?